src/libraries/Common/src/System/Net/SocketAddress.cs

```diff
@@ -20,27 +20,26 @@ namespace System.Net.Internals
 #else
     internal sealed
 #endif
-    class SocketAddress
+    class SocketAddress : System.IEquatable<SocketAddress>
     {
 #pragma warning disable CA1802 // these could be const on Windows but need to be static readonly for Unix
         internal static readonly int IPv6AddressSize = SocketAddressPal.IPv6AddressSize;
         internal static readonly int IPv4AddressSize = SocketAddressPal.IPv4AddressSize;
+        internal static readonly int UdsAddressSize = SocketAddressPal.UdsAddressSize;
+        internal static readonly int MaxAddressSize = SocketAddressPal.MaxAddressSize;
 #pragma warning restore CA1802
 
         internal int InternalSize;
-        internal byte[] Buffer;
+        internal byte[] InternalBuffer;
```

Comment:
You're going to rename these both subsequently, right?

Reply:
yes.